advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>Compared to other SageMaker Notebook examples, it feels somewhat incomplete to end on Training with no Endpoint/Transform inference demonstration. I know that MaskRCNN is primarily considered by the industry as a training benchmarking task, but do you know whether there is any way that the model produced by this training job can make useful predictions. More importantly, can it support inference compatible with SageMaker?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
In general, mask-rcnn model is deployed in embedded systems, such as autonomous vehicles, or surface defect detection instruments. Nevertheless, I plan to create a serving container for inference later. It is just not urgent for this model.

Reply:
OK, got it. Good to hear there may be another iteration in the future. These notebooks are super helpful for showing customers how to use the full suite of SageMaker features.